index.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * Main entry point for hypercerts-lexicon
```

Comment:
Is this also auto-generated?  Just wondering if we have to maintain it manually.